emperor/core.py

```diff
@@ -278,6 +291,31 @@ def make_emperor(self, standalone=False):
         # avoid unicode strings
         coord_ids = list(map(str, self.mf.index.tolist()))
 
+        # TODO: This will be removed once the custom axes creation is moved to
+        # the graphical user interface i.e. to the Axes tab.
+        if custom_axes:
```

Comment:
You are right, but this should be fine, since we can change the axes with the Axes tab right? You could always play around with the order of these.